examples/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pulumi/pulumi-aws-native/examples
 
-go 1.20
+go 1.19
```

Comment:
we should probably update ci-mgmt to not downgrade the go version before applying config generation to this repo